### PR TITLE
Static pages, layout

### PR DESCRIPTION
**Pull request by nhatnguyen71298**
https://edu-redmine.sun-asterisk.vn/issues/34879

- Tạo trang layout cơ bản cho trang home, about,  install gem i18n, bootstrap, jquery.

### CODE REVIEW

**Comment by nhatnguyen71298** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Quản lí chi tiêu cá nhân"
```
Comment (vanvtt-0952) [inline app/helpers/application_helper.rb]: I18n

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```
Comment (vanvtt-0952) [inline app/helpers/application_helper.rb]: `blank?` nè

File: app/views/layouts/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<nav class="navbar navbar-inverse">
+  <%= link_to t(".title"), root_path, class:"navbar-brand"%>
+  <div class="collapse navbar-collapse">
+    <ul class="nav navbar-nav navbar-right">
+      <li><%= link_to t(".home"), root_path %></li>
+      <li><%= link_to t(".about"), page_layout_about_path %></li>
+      <li class="dropdown">
+        <%= link_to "#", class: "dropdown-toggle", "data-toggle": "dropdown" do %>
+        <%= t ".languages" %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_navbar.html.erb]: indent

**Comment by nhatnguyen71298** [comment]
ready

**Pull request merged**